source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,20 +1,97 @@
 ## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#getregister)) and comparing the most recent entry number with your own copy.
+You can use [the `GET /download-register/` endpoint](#download) to download a full new copy of a register.
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+### See how many updates have been made to a register
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#download).
+You can use the [`GET /records/` endpoint](#getrecord) to return all the records from a register. In each record, there is an [`entry-number` property](#entries). The largest value for this is the current size of a register. You can compare this with a local copy. 
 
-Download all updates for one record according to a particular key by [using the `GET /records/{key}/entries` endpoint](#get-record-key-entries):
+Similarly, you can use the [`GET /register/` endpoint](#getregister) to return the `total-entries` property for a register. You can compare this with the same value in a local copy.
 
-Example URL: `https://local-authority-eng.register.gov.uk/records/KIN/entries`
+### Return all entries since your last update
 
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/records/KIN/entries --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
+You can use the [`GET /entries/` endpoint](#getentries) with the optional `start` and `limit` parameters to return specific entries to a register. For example, your highest `entry-number` for your copy of the `country` register could be 205, and you may know that there have been 3 updates to the register since you obtained your copy. You could use the following request, setting `start` to `206` and `limit` to `208`:
```

Comment:
I have the feeling there are quite a few assumptions mixed with the example and with the generic approach. I'm not sure I can explain it better, let me try.

You start by saying that I can use an endpoint and it has two optional parameters. Here I'm not entirely sure it is relevant that they are optional but anyway.

There is again the idea of knowing the highest entry number in my local copy which suggests to me that it should be one of the first recommendations in the "getting updates" section or at least the section that is about getting _just_ the updates not the full set of records.

In the example you say "[...] you may know that there have been 3 updates [...]". To my understanding, the point of using the `GET /entries/` endpoint is to not have to know in advance how many updates have happened. The example could be just about using the `start` parameter with the entry number you (should) know in advance. The result will be all new entries.